### DISCUSSION TITLE
`The command "PATH=$PATH:/usr/local/bin:/opt/homebrew/bin NODE_PATH=`/usr/bin/node /usr/bin/npm root -g` /usr/bin/node '/var/www/laravel/vendor/spatie/browsershot/src/../bin/browser.cjs'

### DISCUSSION THREAD

**Discussion by kinger251285** [spatie/laravel-sitemap#522] [General]
Hi

Im getting the below issue on a brand new install with Laravel 10. I've installed pupetteer via npm. 

`The command "PATH=$PATH:/usr/local/bin:/opt/homebrew/bin NODE_PATH=`/usr/bin/node /usr/bin/npm root -g` /usr/bin/node '/var/www/laravel/vendor/spatie/browsershot/src/../bin/browser.cjs' '{"url":"http:\/\/138.68.164.61\/","action":"content","options":{"args":[],"viewport":{"width":800,"height":600},"executablePath":"\/usr\/bin\/chromium-browser"}}'" failed. Exit Code: 1(General error) Working directory: /var/www/laravel/public Output: ================ Error Output: ================ [1m[43m[30m Puppeteer old Headless deprecation warning:[0m[33m In the near future `headless: true` will default to the new Headless mode for Chrome instead of the old Headless implementation. For more information, please see https://developer.chrome.com/articles/new-headless/. Consider opting in early by passing `headless: "new"` to `puppeteer.launch()` If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.[0m Error: Failed to launch the browser process! spawn /usr/bin/chromium-browser ENOENT TROUBLESHOOTING: https://pptr.dev/troubleshooting at ChildProcess.onClose (/var/www/egn-new/node_modules/@puppeteer/browsers/lib/cjs/launch.js:277:24) at ChildProcess.emit (node:events:513:28) at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12) at onErrorNT (node:internal/child_process:485:16) at processTicksAndRejections (node:internal/process/task_queues:83:21)`

**Comment by Charlilyyyy** [comment]
have u solved it ?
